python-threatexchange/threatexchange/cli/tests/e2e_test_helper.py

```diff
@@ -70,3 +70,13 @@ def assert_cli_usage_error(
         self.assertEqual(exception.returncode, 2)
         if msg_regex is not None:
             self.assertRegex(str(exception), msg_regex)
+
+    def assert_cli_error_output(
+        self, given_args: t.Iterable[str], expected_err: t.Union[str, t.Dict[int, str]]
+    ) -> None:
+        args = list(self.COMMON_CALL_ARGS)
+        args.extend(given_args)
+        print("Calling: $ threatexchange", " ".join(args))
```

Comment:
(blocking): This makes a maintenance challenge, since now the args extension behavior needs to updated in multiple places if we change it. Instead use `self.cli_call()` which will handle it for you.

Reply:
Ahhh good idea. 
What threw me off was the hard-coded "sys.stdout" for my use case and I was averse to changing `self.cli_call()` since it is called in multiple places. My solution was to just create another method 😛 
DRYing this bit with a helper method .